src/raft_server.c

```diff
@@ -739,14 +770,16 @@ int raft_recv_requestvote(raft_server_t* me_,
     {
         e = raft_set_current_term(me_, vr->term);
         if (0 != e) {
+            log_info(me_, node, "rejected requestvote%s for %d: could not update term: %d",
```

Comment:
would this be more appropriate as a log_error()? Since, if the reason e != 0 would occur is because persist_term() callback function (rdb_raft_cb_persist_term) failed, that function will have already called  daos D_ERROR()?

Reply:
Hmm, I don't know why I used `info` here; `error` would be more appropriate indeed.